CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.2.0
```

Comment:
I believe this should be `## Unreleased` for now.

https://github.com/tmuxinator/tmuxinator/blob/master/CONTRIBUTING.md?plain=1#L30

Reply:
thanks fixed